### DISCUSSION TITLE
How to change default configurations of Backbone models ?

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3852] [Q&A]
Hello @artf,
How can I change default config of BlockManager's Category
```js
code from Category.js
 {
  defaults() {
    return {
      id: '',
      label: '',
      open: true,
      attributes: {}
    };
  }
}
```
How to programmatically update open to false ?

**Comment by artf** [comment]
You can get categories (`editor.Blocks.getCategories()`) and update their properties, once blocks are rendered.
